### ISSUE TITLE
rust.js:8-10: Implement rust atom. We need to implement...

### ISSUE THREAD

**Issue by 0pdd** [objectionary/eo2js#62]
The puzzle `3-70345161` from #3 has to be resolved: 

https://github.com/objectionary/eo2js/blob/1b96898e538efd0060ab7bd404982843c72e8447/eo2js-runtime/src/objects/org/eolang/rust.js#L8-L10

The puzzle was created by maxonfjvipon on 22-May-24. 

Estimate: 30 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by maxonfjvipon** [comment]
@levBagryansky could you please help here?

**Comment by R0bari** [comment]
I didn't find EOrust object in provided directory, but I found this: https://github.com/objectionary/eo/issues/3559

It seems EOrust was removed from EO. So I have a question - is this issue still relevant?

**Comment by maxonfjvipon** [comment]
@R0bari you're right, we've removed the `rust` object from `eo-runtime` so the issue is not relevant anymore. The only way to close the ticket - remove todo. You can submit a PR with this change

**Comment by R0bari** [comment]
I'll do it

**Comment by 0pdd** [comment]
The puzzle `3-70345161` has disappeared from the source code, that's why I closed this issue.

**Comment by 0crat** [comment]
@r0bari Thanks for closing this issue! You've earned **+12 points** for this: **+8** as a basis and **+4** for resolving it in less than **12** hours. Your running score is [+12](https://www.eolang.org/zerocracy/objectionary-vitals.html); don't forget to check your [Zerocracy account](https://www.zerocracy.com/) too.
